src/styles/globals.css

```diff
@@ -1,20 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
```

Comment:
In this case, we better use local fonts. One way is that we need to provide the font in the asset directory folder, and if we need it, we just need to install it. 

But for right now, is okay to imports external fonts.